module-4/src/screens/LookSteps/LookSteps.js

```diff
@@ -14,38 +14,48 @@ import Subtitle from '../../components/Subtitle/Subtitle';
 import StepInstructions from '../../components/StepInstructions/StepInstructions';
 import StepInstruction from '../../components/StepInstruction/StepInstruction';
 
-import {ONBOARDING_HOME} from '../../router/routes';
+import {CONGRATULATIONS, ONBOARDING_LOOK_STEPS} from '../../router/routes';
 
 import {color} from '../../styles';
 
 import steps from '../../steps.json';
 
-const LookSteps = ({navigation}) => {
-  const goBackToHome = () =>
-    navigation.navigate(ONBOARDING_HOME, {isStepsCompleted: true});
+const LookSteps = ({route, navigation}) => {
+  const {stepID = 1} = route.params;
```

Comment:
toma cuidado aqui, porque `route.params` pode ser `undefined`, e no caso, causaria um bug do javascript. Uma ideia de dica, que eu costuma fazer pra esses casos, é usar um fallback de objeto, assim:

```suggestion
  const {stepID = 1} = route.params || {};
```